### ISSUE TITLE
setBacklight with 'red' sets blue light on XKE-40

### ISSUE THREAD

**Issue by RubenVerborgh** [SuperFlyTV/xkeys#101]
On the XKE-40, these commands seem to have the same effect:
```
xkeysPanel.setBacklight(0, 'ff0000');
xkeysPanel.setBacklight(0, '0000ff');
xkeysPanel.setBacklight(0, 'ff00ff');
xkeysPanel.setBacklight(0, 'red');
xkeysPanel.setBacklight(0, 'blue');
```

Namely, they set the color of key 0 to blue, whereas I'd expect red or purple in some of the cases.

**Comment by RubenVerborgh** [comment]
Thanks for merging, @nytamin! Would it be possible to issue a patch release? Thanks!

**Comment by nytamin** [comment]
Yep! This is now published in v3.1.2 !
